src/fiber/fiber.mli

```diff
@@ -201,6 +201,34 @@ module Throttle : sig
 end
 with type 'a fiber := 'a t
 
+module Stream : sig
```

Comment:
The name `Sequence` seems a bit more fitting. This is essentially the same type as `Sequence.t` but with the node inside a fiber.

Reply:
Indeed, changing it